components/constellation/constellation.rs

```diff
@@ -782,12 +783,26 @@ impl<Message, LTF, STF> Constellation<Message, LTF, STF>
             .kmerge_by(|a, b| a.instant.cmp(&b.instant) == Ordering::Less)
     }
 
+    #[cfg(not(feature = "unstable"))]
+    /// The joint session future is the merge of the session future of every
+    /// browsing_context, sorted chronologically.
+    fn joint_session_future<'a>(&'a self, top_level_browsing_context_id: TopLevelBrowsingContextId)
```

Comment:
This is incorrect, features are supposed to be additive and this is not.

Reply:
Features being additive is important when one library might be used in the same projects by multiple crate maintained by different people. But the `constellation` crate is not intended to be used outside of this repository. We’ve never worried about backward compat within `components/*`.